src/server/sierraClient/index.ts

```diff
@@ -48,6 +49,21 @@ const sierraClient = async () => {
     })
 
     CACHE.client = wrapper
+    const get = wrapper.get.bind(wrapper)
```

Comment:
In this case, the bind makes more sense with the `await`. You can probably simplify this with `["get", "post", "put"].forEach(method => {...})`